### DISCUSSION TITLE
[Bug]:  Unable to use date property for inline field

### DISCUSSION THREAD

**Discussion by onqun** [RafaelGB/obsidian-db-folder#586] [Q&A]
### Contact Details

ongun@mac.com

### What happened?

I have .md files with bunch of information. including dates (date:: 30/08/2022) İ chose my inline field and then click as calender date nothing shows up on the list

### What platform were you using?

Desktop

### Version of the plugin

2.8.1

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hi @onqun ,
You need to configure your own date format. The default one is `yyyy-MM-dd` . You need `yyyy/MM/dd`


<img width="573" alt="Screenshot 2022-11-08 at 15 15 19" src="https://user-images.githubusercontent.com/11924043/200587740-65080b5b-b261-40a0-bd59-07cfccb99eb3.png">
